### PR TITLE
Suthesh/career page fix

### PR DESCRIPTION
**Pull request by suthesan-deriv**
# Description

Career page layout and content fixes

Changes:

- hide position for locations
- update dubai address
- fix buddhist on ipoh
- fix address line break after comma

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/careers/locations/_location-layout.js [hunk 1]
```diff
@@ -139,16 +139,16 @@ const Iframe = styled.iframe`
...
     if (!display_name) return null
 
-    const positions = getPositionsByLocation(name).sort((a, b) => a.title.localeCompare(b.title))
-    const mapped_positions = positions
-        ? positions.map((position) => ({
-              text: position.title,
-              to: `/careers/jobs/job#${toHashFormat(position.id)}`,
-          }))
-        : []
+    //const positions = getPositionsByLocation(name).sort((a, b) => a.title.localeCompare(b.title))
```
Comment (mustofa-binary) [inline src/pages/careers/locations/_location-layout.js:145]: please add TODO note on commented code

File: src/pages/careers/_model/_locations/_contact-details.js [hunk 2]
```diff
@@ -1,25 +1,43 @@
 import { localize } from 'components/localization'
 
-const address_cyberjaya = localize(
-    'C-13, iTech Tower, Jalan Impact, Cyber 6, 63000 Cyberjaya, Selangor, Malaysia.',
-)
-const address_dubai = localize('1902, Jumeirah Business Center 1, JLT Cluster G, Dubai, UAE.')
+const address_cyberjaya = [
+    localize('iTech Tower, Jalan Impact, Cyber 6,'),
```
Comment (mustofa-binary) [inline src/pages/careers/_model/_locations/_contact-details.js:4]: career page doesnt need localize, applied to all other array localize

**Review by mustofa-binary — CHANGES_REQUESTED** [review]
please also make address changes on contact us page

File: src/pages/careers/locations/_location-layout.js (hunk 1 above)
Comment (suthesan-deriv) [reply src/pages/careers/locations/_location-layout.js:146]: Done 👌

**Review by mustofa-binary — DISMISSED** [review]

File: src/pages/careers/_model/_locations/_contact-details.js (hunk 2 above)
Comment (suthesan-deriv) [reply src/pages/careers/_model/_locations/_contact-details.js]: Done 👌

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**